src/styles/vendors/_icomoon.scss

```diff
@@ -0,0 +1,147 @@
+@use './icomoon-variables' as *;
```

Comment:
The file whcih icomoon generates (which I believe is a .css file) should be used without any changes. This code should be moved to the icomoon-variables file